mappings/lowfat-macula-hebrew.xquery

```diff
@@ -464,6 +465,7 @@ declare function local:node-type($node as element(Node))
             case "pron"
             case "adv"
             case "ij"
+            case "x"
```

Comment:
Some missing `@Cat` values here. Again, my mistake was checking several files and not an XInclude of the entire dataset.